components/EmployeeTableSchedule.js

```diff
@@ -1,28 +1,139 @@
 import React from 'react';
 import Table from 'react-bootstrap/Table';
 
+const dummyData = {
```

Comment:
Please make it a priority to remove this dummy data once the axios calls are set up